vue3_ts/src/layouts/VeevalidateYupLayout.vue

```diff
@@ -0,0 +1,17 @@
+<template>
```

Comment:
Please move this file to the src/layouts/vee-validate-yup/VeevalidateYupLayout.vue